Updating our newsletter strategy (#3047)
PostHog/posthog.com: contents/handbook/growth/marketing/newsletter.md

showTitle: true
---

We publish two newsletters:

1. PostHog Array, a product announcement email published every four weeks

> 2. PostHog News (working title), a marketing and content email published every two weeks

Author: ```suggestion
2. HogMail, a marketing and content email published every two weeks
```

> Both emails are sent via Mailchimp. PostHog News (working title) is a work-in-progress newsletter which we plan to increase to a weekly schedule once we're confident we can produce it to a high standard at that higher frequency. PostHog News is primarily aimed at existing users as a means to improving engagement with our content and aid product discovery through sharing tutorials, tips and ideas, not user acquisition.

Reviewer: Which metric of ours is PostHog News intended to help? I think knowing that will help us focus when in the future we're trying to decide what content to include or not include (e.g. could be Product Discoveries).

Author: Good question. I think influencing discoveries would be hard to attribute. I think the simplest metric is just 'pageviews'. If our users are reading our content, be it tutorials, they'll either be learning something new or engaging more with us as a brand. CTR as measured in Mailchimp (do we have a Mailchimp plug-in?) would be a good quality metric, as wouuld open rates etc.

Author: Yeah, that all makes sense. Discoveries feels like a sensible option.

Reviewer 2: (Good discussion)

I'd have assumed more word of mouth growth would be our goal here.

However, we view this as an expansion of usage exercise?

If so feels like we need way more tutorial stuff in the calendar...

Author: I think it sits in both, really. My view is we can deliver a newsletter that our users enjoy and find useful, it's another means of making them into advocates for us.

Reviewer: > However, we view this as an expansion of usage exercise?

I agree with that - quality content to existing users/newsletter subscribers (who may not be users) should increase their in-org usage, either through recipients spending more time in the product or adding more users within their org. 

@andyvan-ph correct me if I'm wrong, but I believe the intention is to have every newsletter include a tutorial (starting with something every 2 weeks until we're confident we can increase frequency).

Author: Yes, that's correct. I [have a draft](https://docs.google.com/document/d/1hknRyf5Zb6cLuSyEpnawVw6NdRBnva6bQcL3vo37cZw/edit) for comments, though maybe I should make this a PR? Let me know if you'd prefer that showing that.

Basic idea is:

- Newsletter every two weeks
- Every newsletter will feature a blog post and a tutorial
- Some curated content from elsehwhere that we think is interesting and relevant
- A fun cartoon from Lottie despiting a movie/TV show etc. to add some character

But still playing around with it and could really do with some initial impressions.

> Both emails are sent via Mailchimp. PostHog News (working title) is a work-in-progress newsletter which we plan to increase to a weekly schedule once we're confident we can produce it to a high standard at that higher frequency. PostHog News is primarily aimed at existing users as a means to improving engagement with our content and aid product discovery through sharing tutorials, tips and ideas, not user acquisition.

Author: ```suggestion
Both emails are sent via Mailchimp. We plan to move HogMail to a weekly schedule once we're confident we can produce it to a high standard at that higher frequency. It's primarily aimed at existing users as a means to improving engagement with our content and aid product discovery through sharing tutorials, tips and ideas, not user acquisition.
```

## Lists

We currently grow our general email list from three sources:

- Newsletter signups from the website
- PostHog users who opt-in

> - Event attendees

Author: Are we actually doing all of these? From what I can tell, there is no opt-in procedure during account sign-up and I'd be surprised if we actually captured emails from Eventbrite from the recent event.

Feel free to use a few emojis and 1-2 images.

For reference, here's an [example](/blog/the-posthog-array-1-32-0) of a previous newsletter.

> ## PostHog News format (working title)

Author: ```suggestion
## HogMail format
```

The format of the newsletter is still in development, but could/should include the following elements:

- A new piece of fun hedgehog artwork
- Featured content comprising at least one blog post and one tutorial
- Tip/idea of the week to aid discovery  
- PostHog news and product updates, including any updates we've made to documentation

> - Requests for comments on open PRs and issues we'd like feedback on1

Reviewer 3: ```suggestion
- Requests for comments on open PRs and issues we'd like feedback on
```

> The format of the newsletter is still in development, but could/should include the following elements:
> 
> - A new piece of fun hedgehog artwork
> - Featured content comprising at least one blog post and one tutorial
> - Tip/idea of the week to aid discovery  
> - PostHog news and product updates, including any updates we've made to documentation
> - Requests for comments on open PRs and issues we'd like feedback on1

Author: ```suggestion
HogMail roughly follows this format:

- Featured content comprising at least one blog post and one tutorial
- Curated third-party content we think will interest our users 
- PostHog news, feedback requests and community updates
- The Needle comic / cartoon from Lottie
```

> - Community section highlighting and celebrating community contributors
> - Promotion of upcoming events, virtual and in-person

Author: ```suggestion
```
